### ISSUE TITLE
Prevent sliding notification animation in omnibar for autoplay blocks. 

### ISSUE THREAD

**Issue by greetingsj** [brave/brave-browser#3751]
## Description 

My preference is for no site to autoplay, unless I specify otherwise. So I've toggled "Brave Prefs / Content Settings / Autoplay / Ask" to "on". But now whenever I return focus to a tab with autoplay content, a notification slides out the right side of the omnibar, proclaiming "Autoplay was blocked on this page", and then it slides back into the edge of the omnibar. The animation is terribly distracting. 

## Steps to Reproduce

   1. Enable ask-to-autoplay.
   2. Visit any social media page. See it slide out and in. 
   3. Command-shift-bracket for next tab, then repeat to go back. See it slide out and in again. 

## Actual result:

<img width="301" alt="Screen Shot 2019-03-16 at 00 06 27" src="https://user-images.githubusercontent.com/4712441/54472091-2c87e400-4780-11e9-96e0-a7d147daf52e.png">

## Expected result:
No animation. 

## Reproduces how often: 

Always. 

## Brave version (brave://version info)

Version 0.61.51 Chromium: 73.0.3683.75 (Official Build) (64-bit)

### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? 
Unknown. 

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
Yes. 

- Is the issue reproducible on the latest version of Chrome? 
Chrome doesn't have this animation.

**Comment by wakamex** [comment]
this is more annoying than allowing autoplay. it triggers every time i switch channels or tab back to discord.

**Comment by srirambv** [comment]
@rebron I thought this issue was related to #1120  but looks like the user wants to hide the autoplay notification all together unless specified. I do feel this is a little distracting when you visit a site and default action is to stop autoplay. May be we can just show the icon and not the text? I

**Comment by Brave-Matt** [comment]
For what it's worth, I think this message should be present or at least easily displayed whenever autoplay is blocked -- even if it's not in the address bar as it is now. Landing on a page to see all video content "frozen" can often look like an issue rather than the effect a particular setting.

I believe I've actually assisted a user in this exact situation - thinking that there was a web compat. issue with a site they were on when in reality, video and gif content autoplay was blocked and allowing it for the site resolved the issue.

Basically, I think its important that users still have easy access to any/all actions taken by Brave for any site. cc @rebron about moving notifications like this into Shields panel.

**Comment by Brave-Matt** [comment]
Several +1's from Community:
https://community.brave.com/t/autoplay-was-blocked-on-this-page-pops-out-all-the-time/44075/13

**Comment by Brave-Matt** [comment]
+1 from Reddit:
https://www.reddit.com/r/brave_browser/comments/bcqn9v/autoplay_was_blocked_on_this_page_pet_peeve/

**Comment by Brave-Matt** [comment]
+2 more users from Community:
https://community.brave.com/t/autoplay-was-blocked-on-this-page-pop-up/56046/6

**Comment by dchawisher** [comment]
+1 from me. I just switched from Firefox because of the extension issue. The biggest difficulty with the transition has been that irritating notification.

**Comment by nicholasdtc** [comment]
Any news on this? Maybe a simple setting to disable it? It's really getting on my nerves now.

**Comment by npierno** [comment]
+1!!!

**Comment by tanelpoder** [comment]
Adding my +1

**Comment by den1k** [comment]
+1 just installed brave and love it but agree with the sentiments above: this is overly generous notification UX for a mostly insignificant signal

**Comment by Brave-Matt** [comment]
+1 reddit:
https://www.reddit.com/r/brave_browser/comments/bmdazi/remove_autoplay_was_blocked_animation/

**Comment by nigh7m4r3** [comment]
+1 at least remove the animation and keep the full text if needed.

**Comment by hello-jason** [comment]
The [solution here](bitwarden) solves the annoying issue for me.

![image](https://user-images.githubusercontent.com/1406213/58975681-2e63b100-878b-11e9-952d-913ffdd20c0e.png)

**Comment by simonhong** [comment]
@rebron All other content settings also have same issue. Do we only want to hide autoplay animation?

**Comment by rebron** [comment]
I think we're only doing the sliding notification part for autoplay though. All the other content settings don't have a slide out.

**Comment by simonhong** [comment]
No, you can also see slide animation for flash plugin - https://www.ultrasounds.com/

**Comment by rebron** [comment]
@simonhong Ok. Let's take care of it for all cases. Autoplay, flash, any other ones that we're missing?

**Comment by simonhong** [comment]
@rebron  According to the [implementation](https://cs.chromium.org/chromium/src/chrome/browser/ui/content_settings/content_setting_image_model.cc), all content settings model that `blocked_explanatory_text_id` can show slide animation.
They are `flash`, ads`, `automatic download`, `redirect` and `popup`.

**Comment by adamreisnz** [comment]
Glad to see movement on this issue. Just switched to try out Brave, but this message popping up every time I navigated within Intercom was enough to drive me back to Chrome.

To be honest I don't even understand why it's triggering, as there are no video elements on the Intercom page as far as I could see.

**Comment by simonhong** [comment]
@adamreisnz I think below that page has some video elements like below.
<img width="1638" alt="Screen Shot 2019-06-11 at 10 52 53 AM" src="https://user-images.githubusercontent.com/6786187/59237970-29917800-8c37-11e9-8f3b-7cca17fb87ce.png">

**Comment by adamreisnz** [comment]
@simonhong it's not on the intercom website, it's inside the app. I've looked around the page and source, but there are no video's there as far as I can see.

**Comment by sanford** [comment]
Is this in 65.121??  Still getting this message in every Reddit tab I scroll (multiple times per scroll).

**Comment by the-4-limbos** [comment]
Brave is up to date
Version 0.66.99 Chromium: 75.0.3770.100 (Official Build) (64-bit)

it is still in this version. Any ETA on when the sliding notification will be removed and replaced with an icon like the shield button next to it?

thank you and keep up the good work :)

**Comment by simonhong** [comment]
@sanford @the-4-limbos It is in dev channel (0.68.x) now :)
We are planning to release 0.68 as a stable at August 20, 2019.
Thanks for checking!

**Comment by btlechowski** [comment]
Verification passed on

Brave | 0.68.120 Chromium: 76.0.3809.87 (Official Build) beta (64-bit)
-- | --
Revision | 111fe1e15d5ced26080a7dc239bcfe70f6c49aad-refs/branch-heads/3809@{#967}
OS | Windows 10 OS Version 1809 (Build 17763.379)

Logged, a serious regression https://github.com/brave/brave-browser/issues/5605
